blog/templates/blog/blogbase.html

```diff
@@ -227,7 +227,7 @@ <h2 class="blog-post-title">New feature</h2>
     ================================================== -->
     <!-- Placed at the end of the document so the pages load faster -->
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.2/jquery.min.js"></script>
-    <script src="../../dist/js/bootstrap.min.js"></script>
+    <script src="{% static 'bootstrap/js/bootstrap.min.js' %}"></script>
     <!-- IE10 viewport hack for Surface/desktop Windows 8 bug -->
     <script src="../../assets/js/ie10-viewport-bug-workaround.js"></script>
```

Comment:
For any files or images, if it was just copy and paste code and you didn't actually copy the files over, then remove them.